autosubmit/autosubmit.py

```diff
@@ -767,11 +773,13 @@ def parse_args():
             return Autosubmit.update_version(args.expid)
         elif args.command == 'upgrade':
             return Autosubmit.upgrade_scripts(args.expid,files=args.files)
+        elif args.command == 'provenance':
+            return Autosubmit.provenance(args.expid, rocrate=args.rocrate)
         elif args.command == 'archive':
-                return Autosubmit.archive(args.expid, noclean=args.noclean, uncompress=args.uncompress, rocrate=args.rocrate)
+            return Autosubmit.archive(args.expid, noclean=args.noclean, uncompress=args.uncompress, rocrate=args.rocrate)
         elif args.command == 'unarchive':
             return Autosubmit.unarchive(args.expid, uncompressed=args.uncompressed, rocrate=args.rocrate)
-
+        
```

Comment:
Unnecessary spaces :grimacing: 

The next time someone opens this file, their editor might remove this automatically (PyCharm does that, and VS Code has a way of doing this as well [Eclipse did that doo]). Which means it would pop up in the pull request of somebody else (i.e. simpler to undo it here :+1: )